org.eclipse.gef.examples.text/src/org/eclipse/gef/examples/text/TextEditor.java

```diff
@@ -78,6 +78,35 @@ public class TextEditor extends GraphicalEditor {
 	private Container doc;
 	private StyleService styleService = new StyleService();
 
+	private static final class TextEditorEditPartFactory implements EditPartFactory {
+		@Override
+		public EditPart createEditPart(EditPart context, Object model) {
+			if (model instanceof Container cont) {
+				switch (cont.getType()) {
+				case Container.TYPE_ROOT:
+					return new DocumentPart(cont);
+				case Container.TYPE_IMPORT_DECLARATIONS:
+					return new ImportsPart(cont);
+
+				case Container.TYPE_COMMENT, Container.TYPE_PARAGRAPH:
```

Comment:
Minor formatting issue

Reply:
How would you format this? I thought the new case statement with commas between cases is formated on one line.